content/swiftui/concepts/views/terms/lazyVStack/lazyVStack.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Lazy VStack'
+Description: 'A view that arranges its children in a line that grows vertically, creating items only as needed.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+
+The **`Lazy VStack`** has all the properties as a standard VStack **BUT** it will not create items until it needs to render them on screen. This comes very handy when creating long lists and your app can run smoother and faster.
+
+## Syntax
+
+```pseudo
+var body: some View {
+    LazyVStack {
+        Subviews here
+    }
+}
+```
+
+The `LazyVStack` view rests within the body of a `View`.
+
+## Example
+
+In this example a `ScrollView` contains a `LazyVStack` that consists of a vertical row of text views. The `LazyVStack` will create the rows only when they are needed.
```

Comment:
```suggestion
In the example below, a `ScrollView` contains a `LazyVStack` that consists of a vertical row of text views. The `LazyVStack` will create the rows only when they are needed.
```

Since there is yet to be an entry on scroll views, can we include a sentence explaining about it? Also, it may be good to include the last sentence's explanation up on line 24 after the pseudo code block.